### PR TITLE
Update TTL Manager

### PR DESCRIPTION
**Pull request by mjangda**
Drops the class and updates docs.

See #673

### CODE REVIEW

File: vip-cache-manager/ttl-manager.php [hunk 1]
```diff
@@ -2,50 +2,59 @@
...
+	return $response;
+}
+
+function set_rest_response_ttl( \WP_REST_Response $response, $ttl ) {
+	$response->header( 'Cache-Control', sprintf( 'max-age=%d', $ttl ) );
 }
 
-TTL_Manager::init();
+// Let's do it!
+init();
```
Comment (joshbetz) [inline vip-cache-manager/ttl-manager.php]: We could probably even call `add_filter()` directly here?

Comment (mjangda) [inline vip-cache-manager/ttl-manager.php]: Keeping it in `init` to allow for future, additional TTL overrides we may want to add.

File: vip-cache-manager/ttl-manager.php [hunk 2]
```diff
@@ -2,50 +2,59 @@
 
 /*
 Plugin name: TTL Manager
-Description: Sets sane and reasonable cache TTLs for site responses.
+Description: Sets cache TTLs for site responses, see https://vip.wordpress.com/documentation/vip-go/controlling-vip-go-page-cache/
```
Comment (simonwheatley) [inline vip-cache-manager/ttl-manager.php]: Last night it occurred to me we should add more links 🖖

Comment (mjangda) [inline vip-cache-manager/ttl-manager.php]: Pretty sure [this was your idea](https://github.com/Automattic/vip-go-mu-plugins/pull/672#discussion_r125284689) :)

**Comment by mjangda** [comment]
r95041-deploy

File: vip-cache-manager/ttl-manager.php (hunk 2 above)
Comment (simonwheatley) [inline vip-cache-manager/ttl-manager.php]: 🙃

**Pull request merged**